src/reducers/index.js

```diff
@@ -14,6 +16,8 @@ const rootReducer = combineReducers({
   environments,
   errors,
   repositories,
+  contributors,
```

Comment:
Wait, I think you are misinterpreting the use of Contributors: `Contributors` are those people who have contributed to at least one out of three fixed repositories: `scigym`, `scigym-web` or `scigym-api`. And as of now, we are in charge of updating the `Contributors` manually, by calling `find_contributors`. Eventually, we should automate this and make this API call every other day or so. This automatically updates the list of contributors. Since the number of repos which is parsed in order to find the `Contributors` is constant, I don't think that we get any overhead. `ProjectAuthors` is just a (for now) fixed number of people, i.e. you, I and 2 others we started this project with. The `Contributors` then appear in the `Impressum` to show who has helped to built SciGym.